$NetBSD: patch-aa,v 1.2 2006/03/12 13:47:36 rillig Exp $

The paths to perl and awk need to be set explicitly, otherwise the ones
from the .tools directory are found. The rest of the patch adjusts the
directories to the pkgsrc directory hierarchy layout.

--- Makefile.orig	2003-06-14 19:38:45.000000000 +0200
+++ Makefile	2006-03-12 13:50:56.000000000 +0100
@@ -1764,8 +1764,8 @@ installtools: $(TOOLS) $(DESTDIR)$(exec_
 	      fi
 	-chmod $(FILEMOD) $(DEST_TOOLS)/*
 # replace the path in some tools
-	perlpath=`./which.sh perl` && sed -e "s+/usr/bin/perl+$$perlpath+" $(TOOLSSOURCE)/efm_perl.pl >$(DEST_TOOLS)/efm_perl.pl
-	awkpath=`./which.sh nawk` && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; if test -z "$$awkpath"; then \
+	perlpath='$(PERL5)' && sed -e "s+/usr/bin/perl+$$perlpath+" $(TOOLSSOURCE)/efm_perl.pl >$(DEST_TOOLS)/efm_perl.pl
+	awkpath='$(AWK)' && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; if test -z "$$awkpath"; then \
 		awkpath=`./which.sh gawk` && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; if test -z "$$awkpath"; then \
 		awkpath=`./which.sh awk` && sed -e "s+/usr/bin/nawk+$$awkpath+" $(TOOLSSOURCE)/mve.awk >$(DEST_TOOLS)/mve.awk; fi; fi
 	-chmod $(SCRIPTMOD) `grep -l "^#!" $(DEST_TOOLS)/*`
@@ -1789,19 +1789,19 @@ install-languages: languages $(DEST_LANG
 # doesn't.<=looks strange to install icons to user's directory ;), i add my
 # own install procedure, mikmak
 installkdeicons:
-	mkdir -p $(DESTDIR)$(KDE_DIR)/share/applnk/Editors/
-	mkdir -p $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/32x32/apps/
-	mkdir -p $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/48x48/apps/
-	mkdir -p $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/22x22/actions/
-	mkdir -p $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/16x16/actions/
-	mkdir -p $(DESTDIR)$(KDE_DIR)/share/apps/kvim
-
-	cp ../runtime/KVim.desktop $(DESTDIR)$(KDE_DIR)/share/applnk/Editors/ && chmod 644 $(DESTDIR)$(KDE_DIR)/share/applnk/Editors/KVim.desktop
-	cp ../runtime/kvim32x32.png $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/32x32/apps/kvim.png && chmod 644 $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/32x32/apps/kvim.png
-	cp ../runtime/kvim48x48.png $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/48x48/apps/kvim.png && chmod 644 $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/48x48/apps/kvim.png
-	cp ../runtime/hi16-action-make.png $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/16x16/actions/hi16-action-make.png
-	cp ../runtime/hi22-action-make.png $(DESTDIR)$(KDE_DIR)/share/icons/hicolor/22x22/actions/hi22-action-make.png
-	cp ../runtime/kde-tips $(DESTDIR)$(KDE_DIR)/share/apps/kvim/tips && chmod 644 $(DESTDIR)$(KDE_DIR)/share/apps/kvim/tips
+	mkdir -p $(DESTDIR)$(KDE_DIR)/share/kde/applnk/Editors/
+	mkdir -p $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/32x32/apps/
+	mkdir -p $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/48x48/apps/
+	mkdir -p $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/22x22/actions/
+	mkdir -p $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/16x16/actions/
+	mkdir -p $(DESTDIR)$(KDE_DIR)/share/kde/apps/kvim
+
+	cp ../runtime/KVim.desktop $(DESTDIR)$(KDE_DIR)/share/kde/applnk/Editors/ && chmod 644 $(DESTDIR)$(KDE_DIR)/share/kde/applnk/Editors/KVim.desktop
+	cp ../runtime/kvim32x32.png $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/32x32/apps/kvim.png && chmod 644 $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/32x32/apps/kvim.png
+	cp ../runtime/kvim48x48.png $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/48x48/apps/kvim.png && chmod 644 $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/48x48/apps/kvim.png
+	cp ../runtime/hi16-action-make.png $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/16x16/actions/hi16-action-make.png
+	cp ../runtime/hi22-action-make.png $(DESTDIR)$(KDE_DIR)/share/kde/icons/hicolor/22x22/actions/hi22-action-make.png
+	cp ../runtime/kde-tips $(DESTDIR)$(KDE_DIR)/share/kde/apps/kvim/tips && chmod 644 $(DESTDIR)$(KDE_DIR)/share/kde/apps/kvim/tips
 
 ICON48PATH = $(DESTDIR)$(DATADIR)/icons/hicolor/48x48/apps
 ICON32PATH = $(DESTDIR)$(DATADIR)/icons/locolor/32x32/apps
